File: Armaan4pm/MyFirstApp/app/src/main/java/com/example/myfirstapp/SecondActivity.java
package com.example.myfirstapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class SecondActivity extends AppCompatActivity {
TextView emailtview;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
        emailtview = findViewById(R.id.thirdext);

        String values = getIntent().getStringExtra("email");
        emailtview.setText(values);
    }
}

File: Rahoul3pm/RadioButtonsecond/app/src/main/java/com/example/radiobuttonsecond/SecondActivity.kt
package com.example.radiobuttonsecond

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class SecondActivity : AppCompatActivity() {
    lateinit var  mytextview:TextView
    lateinit var emailvalue:String
    lateinit var namevalue : String
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        mytextview = findViewById(R.id.firstextview)
       emailvalue= intent.getStringExtra("email")!!
        namevalue = intent.getStringExtra("name")!!


        mytextview.text = " $emailvalue \n $namevalue"
    }
}